assets/data/majors.json

```diff
@@ -51,7 +51,10 @@
   "Communication Technologies",
   "Computer and Information Systems",
   "Computer Programming and Data Processing",
-  "Computer Science",
+  "Computer Science (CSCI)",
```

Comment:
Added these for better autocomplete, also catering to alt-CS majors at USC 

(I can imagine a lot of people typing in `CS` or `CSBA` for major, autocomplete will show these as suggestions if they do so)